src/data/teams.json

```diff
@@ -641,6 +641,19 @@
           "profileImage": "https://avatars0.githubusercontent.com/u/32864116?s=460&u=0c2a53c6f1476da904d2b3c820bab43b64b7185b&v=4",
           "slackUsername": "James Pang"
         }
+      },
+      {
+        "title": "WordPress Developer",
+        "isLead": false,
+        "payPerDay": 2800,
+        "createdDate": "2020-07-08T02:14:59.307535Z",
+        "contributor": {
+          "contributorId": "22",
```

Comment:
Should be `67` after https://github.com/thenewboston-developers/Website/pull/1202?

Reply:
Alright.

Reply:
Please take note that it should be the largest `contributor id + 1` across all teams, not the previous `contributor id + 1`

Reply:
Noted